content/docs/courses/fundamentals/zero-custom-idp.mdx

```diff
@@ -0,0 +1,81 @@
+---
+id: zero-custom-idp
+title: Custom Identity Provider
+description: In this guide, learn how to configure a custom identity provider in the Zero Console.
+sidebar_label: 07. Custom Identity Provider
+sidebar_position: 7
+---
+
+# Zero Fundamentals: Configure a Custom Identity Provider
+
+So far, you've been using our [Hosted Authenticate service](/docs/capabilities/hosted-authenticate-service), which provides a hosted authenticate service URL (`authenticate.pomerium.app`) and Pomerium's hosted identity provider to authenticate users.
+
+In this guide, you'll replace Pomerium's hosted identity provider with your own custom identity provider to handle authentication.
+
+:::note Before you start
+
+Make sure you've completed the following tutorials:
+
+- [**Build a Simple Route**](/docs/courses/fundamentals/zero-build-routes)
+- [**Build a Simple Policy**](/docs/courses/fundamentals/zero-build-policies)
+- [**Single Sign-on with JWTs**](/docs/courses/fundamentals/zero-single-sign-on)
+- [**Build Advanced Policies**](/docs/courses/fundamentals/zero-advanced-policies)
```

Comment:
I'm not sure these are strictly necessary for this guide , but I don't have strong opinions either way.